diesel_tests/tests/internal_details.rs

```diff
@@ -1,6 +1,6 @@
 use schema::*;
 use diesel::sql_types::*;
-use diesel::dsl::sql;
+use diesel::dsl::{sql, select};
```

Comment:
Why do we import select here and then use a qualified call to `insert_into` below.
(In general I would vote for having more imports and better remove the qualified calls in our test suite. This means instead of doing `select(something)` → `dsl::select(something)` better do one `use diesel::dsl::select` on the top of the file. This applies for all changes in diesel_test and diesel_compile_test)